includes/compat/class-wc-stripe-subscriptions-helper.php

```diff
@@ -24,26 +24,30 @@ public static function is_subscriptions_enabled() {
 	}
 
 	/**
-	 * Returns a list of subscriptions that are detached from the customer.
+	 * Returns a list with some (up to 5) subscriptions that are detached from the customer.
```

Comment:
(nit) This doc string can be easy to misinterpret. This function does not look for up to 5 detached subscriptions. Rather, it looks at 5 subscriptions and checks if they are detached.

Reply:
You're right. On https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4067/commits/113b978276e3256ef44a2c6db524a2b3a182c70f I have increased the number of subscriptions queried to 50 and added the breaking condition in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4067/commits/9f48635d69417b761885b91b51a28c2994a391a3, so we have better chances of getting the maximum of 5 items to notify.

I tried to change the logic to query using the `_stripe_customer_id` directly to get five subscriptions more consistently, but that was too slow.